src/main/java/it/gov/pagopa/payment/dto/qrcode/SyncTrxStatusDTO.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.payment.dto.qrcode;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class SyncTrxStatusDTO {
+    private String id;
+    private String idTrxIssuer;
+    private String trxCode;
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    private OffsetDateTime trxDate;
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    private LocalDateTime authDate;
+    private String operationType;
```

Comment:
The type of this field should be src/main/java/it/gov/pagopa/payment/enums/OperationType.java

Reply:
It depends on whether we want to return the operation type "00"/"01" or "CHARGE"/"REFUND".  The latter should require the enum. It is not mandatory, but it is safer

Reply:
00 or 01 are something inner to rewardCalculator, here let's use the enum ;)

Reply:
This field in TransactionInProgress is a String. Do we want to replace them in an enum as well?

Reply:
TransactionInProgress#operationType is something else!
we should mapping SyncTrxStatus#operationType with TrxInProgress#operationTypeTranscoded

If inside the mapper you are mapping TrxInProgress#operationType, that is an error not seen by us :)